src/ccl_cls.c

```diff
@@ -369,6 +371,11 @@ static CCL_ClTracer *cl_tracer_new(ccl_cosmology *cosmo,int tracer_type,
       }
     }
   }
+  else if(tracer_type=CL_TRACER_CL) {
```

Comment:
I think you need a "==" instead of "=" here. The compiler gave me a warning.

Reply:
Well spotted! Done.